alchemy-web/src/content/docs/guides/cloudflare-nuxt.mdx

```diff
@@ -233,16 +231,31 @@ The CLI updated the `tsconfig.json` to include `alchemy.run.ts` and register `@c
 
 ### `nuxt.config.ts`
 
-Use `@nuxthq/studio` and `@nuxtjs/cloudflare` to build for Cloudflare Workers:
+Use the `cloudflare-module` preset and `nitro-cloudflare-dev` module to build for Cloudflare in production and development:
 
 ```typescript
+import alchemyCloudflare from "alchemy/cloudflare/nuxt";
+import { defineNuxtConfig } from "nuxt/config";
+
 export default defineNuxtConfig({
-  modules: ["@nuxthq/studio", "@nuxtjs/cloudflare"],
+  compatibilityDate: "2025-04-21",
```

Comment:
Can alchemy plugin set this? Ideally it would come from the website resource definition.